website/content/about/working-groups/contributor-growth.md

```diff
@@ -4,18 +4,111 @@ linkTitle: Contributor Growth
+    * Support their maintainers with advice and resources that improve sustainability and plan for transitions from the project.
+
+{{% /blocks/section %}}
+
+</div>
+
+<div class="section-group">
+{{< blocks/section color="primary" >}}
+
+## How your community can benefit
+
+Here are some benefits that the Contributor Growth working group can provide to communities:
+
+* Best practices and resources for managing and interacting with contributors to CNCF projects
+* Guidance and support for a sustainable contributor lifecycle for CNCF projects
+* Templates for common documents like a contributing guide, reviewing guide, TOC due diligence guidelines, and more
+* Case studies and talks for insights and inspiration
+* Non-code areas such as project management, work prioritization and similar, which may not be the maintainers' primary skillset
+
+You may find some tips in our [Community Guide](/maintainers/community/) and [Available Templates](/maintainers/templates/).
+
+{{% /blocks/section %}}
+</div>
+
+<div class="section-group">
+{{< blocks/section color="primary" >}}
+
+## How you can contribute
+
+We are happy to see that you are interested in helping with this working group. We would love to:
+* get your feedback about out guides and templates.
+* see your introduction email on our mailing list / message on our Slack channel.
+* see you attend our meetings where you can contribute ideas and take responsibilities.
+
+{{% /blocks/section %}}
+</div>
+
+<div class="section-group">
+
+## Get involved
+
+{{< blocks/section color="primary" >}}
+
+{{% blocks/feature title="Meetings" icon="fas fa-video" %}}
+
+<div>
+
+View our [calendar](https://tockify.com/cncf.public.events/monthly?search=Contributor%20Growth%20WG) to see when we meet next.
+
+<a href="https://docs.google.com/document/d/1Kx7tZv5wTXQ7uRKxn5d9d2wLsI3Q3Q51A0i06nLvtdI/edit">Meeting minutes and agenda</a>
+<a href="https://zoom.us/my/cncftagcontributorstrategy?pwd=TnI0WU9Eb2I1RlRWdkl1R0k1WkZXUT09">Meeting Link</a>
+
+</div>
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fas fa-envelope" title="Mailing List"
+url="https://lists.cncf.io/g/cncf-tag-contributor-strategy" url_text="Sign Up"
+%}}
+Keep up-to-date on mentoring programs we manage.
+
+{{% /blocks/feature %}}
+{{< /blocks/section >}}
+
+{{< blocks/section color="gray-300" >}}
+
+{{% blocks/feature title="Slack" icon="fab fa-slack" %}}
+Reach out for any immediate matters at [#tag-contributor-strategy](https://cloud-native.slack.com/archives/CT6CWS1JN) on [CNCF Slack](https://slack.cncf.io)
+
+{{% /blocks/feature %}}
+
+{{% blocks/feature icon="fas fa-info" title="Find out more"
+url="https://github.com/cncf/tag-contributor-strategy/blob/main/contributor-growth/README.md" url_text="Charter Document"
+%}}
+More resources are available on our Charter document, [Community Guide](/maintainers/community/) and [Available Templates](/maintainers/templates/).
+{{% /blocks/feature %}}
```

Comment:
I recommend removing the link from the block and instead linking the word charter instead in "available on our Charter document". Right now it looks a bit disjointed
![Screen Shot 2023-04-17 at 10 01 25 AM](https://user-images.githubusercontent.com/1368985/232526111-b41ae2da-3b89-42ae-82ac-fc1a7bfa8a9b.png)

Reply:
👍